src/helpers/limit_results.js

```diff
@@ -0,0 +1,8 @@
+const limitResults = (results) => {
+  while (results.length != 9 && results.length > 9) {
```

Comment:
cause i wanted to limit it to 9 but i guess now im going to change it cause we dont want any limitation